.gitignore

```diff
@@ -74,7 +74,6 @@ gradle-app.setting
 *.zip
 *.tar.gz
 *.rar
-
```

Comment:
Also remove the new line :D

Reply:
oh what that dosent show in my ide?

Reply:
> Also remove the new line :D

it was github actualy that dosent exist

Reply:
Try to remove the file and then just add it back by download it from Snowiiii master